### PR TITLE
Aws setup scripts

### PR DESCRIPTION
**Pull request by SushilSanjayBhile**
1) Readme file with steps to deploy all resources

2) aws.env file which contains all environment variables, same values of which like BACKEND_VPC_NAME, BACKEND_VPC_REGION are needed in different scripts.

3) Backend VPC creation:
    i)  Bash script (create-backend-vpc-cloudformation-stack.sh)
    ii) Cloudformation template (backend-vpc.yaml)

4) Controller VPC creation:
    i)  Bash script (create-controller-vpc-cloudformation.sh)
    ii) Cloudformation template (controller-vpc.yaml)

5) Vault creation:
    i) Vault SSL certificate creating bash script & Cloudformation stack (create-vault-ssl-certificate-cloudformation-stack.sh & ssl-certificate-template.yaml)
    ii) Vault cluster Cloudformation stack (hashicorp-vault-cloudformation-spec.yaml)

6) Postgres RDS creation
    i)  Bash script (create-postgres-rds-cloudformation.sh)
    ii) Cloudformation template (postgres-rds-cloudformation.yaml)

7) VPC peering:
    i)  Bash script (create-backend-controller-vpc-pairing.sh)
    ii) Cloudformation template (vpc-pairing-cloudformation.yaml)

### CODE REVIEW

**Comment by SushilSanjayBhile** [comment]
@sraghavan-drio Can you please review the Readme file and scripts?

**Comment by SushilSanjayBhile** [comment]
Also, @sraghavan-drio I am planning to restructure the AWS directory into something like the following:

aws/
-    Root README
-    aws.env 

-    vault/
      - vault-cloudformation templates (SSL creation and hashi-corp template)
      - vault scripts
      - Readme

-    Backend_VPC/
      - cloud formation templates
      - bash scripts
      - Readme

-    Controller_VPC/
      - cloud formation templates
      - bash scripts
      - Readme

-    Postgres
      - cloud formation templates
      - bash scripts
      - Readme

-    other scripts/
      - VPC pairing script
      - Readme

File: aws/README.md
Comment (sraghavan-drio) [inline aws/README.md:34]: What is MASTER_USER_PASSWORD? Is it for AWS or Postgres?

File: aws/README.md
Comment (sraghavan-drio) [inline aws/README.md:42]: Is it VPC Peering and not Pairing

**Review by sraghavan-drio — COMMENTED** [review]
@SushilSanjayBhile,

Looks good. Please respond to the comments and I think we can merge this PR after that.

File: aws/README.md
Comment (SushilSanjayBhile) [reply aws/README.md:34]: It is for postgres user. I will rename this variable to POSTGRES_MASTER_USER_PASSWORD

File: aws/README.md
Comment (SushilSanjayBhile) [reply aws/README.md]: Renamed script and README with typo fix

**Comment by SushilSanjayBhile** [comment]
> @SushilSanjayBhile,
> 
> Looks good. Please respond to the comments and I think we can merge this PR after that.

Hi @sraghavan-drio, I have taken care of the above comments.
Can you Approve the PR?

**Review by sraghavan-drio — APPROVED** [review]
Looks good.

**Comment by SushilSanjayBhile** [comment]
Hi @sraghavan-drio Looks like I don't have write access, so I am unable to merge.

Message in PR:
"Only those with [write access](https://docs.github.com/articles/what-are-the-different-access-permissions) to this repository can merge pull requests."

**Pull request merged**
